openrtb_ext/bidders.go

```
@@ -94,11 +93,11 @@ func NewBidderParamsValidator(schemaDirectory string) (BidderParamValidator, err
 		if _, isValid := BidderMap[bidderName]; !isValid {
 			return nil, fmt.Errorf("File %s/%s does not match a valid BidderName.", schemaDirectory, fileInfo.Name())
 		}
-
-		schemaLoader := gojsonschema.NewReferenceLoaderFileSystem(fmt.Sprintf("file:///%s", fileInfo.Name()), filesystem)
+		toOpen, _ := filepath.Abs(filepath.Join(schemaDirectory, fileInfo.Name()))
```

This should probably return the error, rather than ignoring it